README.md

```diff
@@ -60,12 +59,20 @@
   </ul>
 </details>
 
+### Key Features <a name="key-features"></a>
+
 <details>
   <summary>Styling</summary>
   <ul>
     <li><a href="https://expressjs.com/">CSS</a></li>
   </ul>
 </details>
+<details>
+  <summary>React</summary>
+  <ul>
+    <li><a href="https://expressjs.com/">CSS</a></li>
+  </ul>
+</details>
 <details>
   <summary>Functionality</summary>
```

Comment:
You have mixed up the content of the key features and tech stark please make sure that the content for each section matches the heading of the section 👍🏿